Directory.Build.props

```diff
@@ -77,7 +77,7 @@
     -->
     <Otherwise>
       <PropertyGroup>
-        <DefaultNetCoreTargetFramework>net8.0</DefaultNetCoreTargetFramework>
+        <DefaultNetCoreTargetFramework>net9.0</DefaultNetCoreTargetFramework>
```

Comment:
Hm, I guess it's not that simple, parts that deploy to VS/VSCode probably need to stay on net8.0.

Reply:
To make it more of a gradual change, you could add net9 to the multi-targeting, and we can continue to change it as the needs of SDK/VS/VSCode change.  There are some tradeoffs to making the multi-targeting conditional to being on a CI machine or not.